tests/unit/test_actions.py

```diff
@@ -0,0 +1,130 @@
+import dataclasses
```

Comment:
Can you please add the copyright ?

Reply:
Added in 242884c4150cb43cdda7eb84fdcfe78ec7b43016